### ISSUE TITLE
[Adaptive Cards- Blog] : While navigating using arrow keys, On pressing down arrow key, blog link is getting activated automatically. 

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#9030]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/
OS Version: Dev (OS Build 27695.1000)
Browser Version: Version 130.0.2849.1 (Official build) dev (64-bit)

**Pre-Requisite:**
Navigate through keyboard only.

**Repro steps:**
1. Open the above URL in the edge dev browser.
2. Navigate to the 'Blog' option and press enter key to activate it.
3. Press tab key and navigate to the blog.
4. Press tab key and navigate to any blog.
5. Press right arrow key and observe the issue.

**Actual Result:**
While navigating using down arrow key, Blog links are getting automatically activated.

**Observation:**
While navigating using Left/Right/Up arrow key, this issue is working fine.

**Expected Result:**
While navigating the Blogs using arrow keys, on pressing down arrow key blog should be not activated instead of activating the blog keyboard focus should be land on the same.

**User Impact:**
It will impact the keyboard user as if the blogs will be open automatically as it will consume their time to renavigate on the same page.

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

![While navigating using arrow keys, On pressing down arrow key, blog link is getting activated automatically](https://github.com/user-attachments/assets/42023098-62bd-4376-97cb-4431eb452a35)

https://github.com/user-attachments/assets/a8d6ef4f-fd3c-42ac-816a-3fd4025e1007

### Card JSON

```JSON
NA
```
